nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Tag management pm

# Description
Added Tag handling features.
## Type of change
Please delete options that are not relevant.


# Testing Instructions
While logged in, the Tag Management NavBar link will be available. The list of all available tags will be displayed as a list in 'Button' format. Clicking any of the tags will open an edit view in place. The Edit window allows for the changing of the tag name, or deleting the tag itself. Once a change is made to the text of the tag name, a 'Save' button will become visible allowing you to save your changes. A 'discard' button is always visible which allows the user to cancel the action at any point. Clicking Delete will open a confirm window in place allowing you to cancel out or complete the delete operation.

CSS is in place to change the view order on smaller devices. The edit window will appear above the tag list below screen width 480px.

(No Add in place at this moment)
# Checklist:
